src/main/groovy/com/marklogic/gradle/MarkLogicPlugin.groovy

```diff
@@ -249,6 +250,9 @@ class MarkLogicPlugin implements Plugin<Project> {
 			"Use -ProxyProjectPath to define the location of your Roxy project.")
 		project.task("mlRoxyMigrateProject", group: roxyGroup, description: "Run all tasks for migrating a Roxy project into this Gradle project. " +
 			"Use -ProxyProjectPath to define the location of your Roxy project.", dependsOn: ["mlRoxyMigrateBuildSteps", "mlRoxyMigrateFiles", "mlRoxyMigrateProperties"])
+
+		String testGroup = "ml-gradle Test";
+		project.task("mlRoxyTest", type: RoxyTest, group: testGroup, description: "Run Roxy Unit Tests", dependsOn: ["mlDeployApp", "mlPostDeploy"])
```

Comment:
I think it'd be helpful for the description to include the property that it depends on. If the property isn't defined, Gradle will error out. A nice thing to do is to fail gracefully if the property doesn't exist, and then print out the task description - see RemoveCollectionsTask or any of the DMSDK tasks for an example.